### ISSUE TITLE
Opção de Zero do Primeiro Dígito Logístico como reserva

### ISSUE THREAD

**Issue by ppKrauss** [osm-codes/gridMap-draftPages#90]
Nesta proposta cada país teria **mais uma decisão** soberana: manter nos municípios o primeiro dígito zero como recurso de "reserva mínima obrigatória", fascilitando a comunicação nacional, para usuários estarem cientes de que o município lançou mão da reserva.

## Implementação 
Permitir que na listagem de coberturas compareça o NULL para a contagem, ou que cada município tenha seu flag de "cover_ignore_zero".

**Comment by 0e1** [comment]
Implementando, conforme os commits referenciados acima.

Cobertura de Tunja atualizada. Ver: https://afa.codes/CO-15001.

**Comment by 0e1** [comment]
Considerando https://github.com/osm-codes/AFAcodes-softwareGen/issues/31 e que o índice atribuído as células das coberturas farão parte do git ([exemplo em CM](https://github.com/osm-codes/CM/blob/main/data-restricted/citycover.csv)) essa issue e o que foi implementado nela ficou obsoleto e não sera mais usado.

**Comment by ppKrauss** [comment]
exato, ignorar essa issue 90, o que vale é a grade de fundo, a grade de referência, incluindo os zeros dela. As reservas terão códigos aleatórios e tudo bem
